### PR TITLE
Getting Started Cleanup

### PR DESCRIPTION
**Pull request by JoelMarcey**
1. Remove note about upgrading Homebrew packages. That was kind of noisy.
2. Use local images instead of those stored on Akamai.
3. Add wording for modifying test project about actually opening a file.
4. Add note about keeping initial defaults for Android Studio install in tact.

Tested site locally. Images and new wording appeared as expected.

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
By analyzing the blame information on this pull request, we identified @willfong and @DavidYKay to be potential reviewers.

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ block { display: none; }
 
 <div>Unfortunately, Apple only lets you develop for iOS on a Mac machine. Please check out the <a href="" onclick="display('platform', 'android')">Android</a> instructions instead.</div>
 
-<center><img src="https://fbcdn-dragon-a.akamaihd.net/hphotos-ak-xaf1/t39.1997-6/851591_233289256829505_447005964_n.png" width="150"></img></center>
+![not supported](img/react-native-sorry-not-supported.png)
```
Comment (vjeux) [inline docs/GettingStarted.md]: you are missing the `<center>` and width

```
<center><img src="img/react-native-sorry-not-supported.png" width="150"></img></center>
```

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -105,14 +103,20 @@ npm install -g react-native-cli
...
 
 > Android Studio requires the Java Development Kit [JDK] 1.8 or higher. You can type
 > `javac -version` to see what version you have, if any. If you do not meet the JDK requirement,
 > you can
 > [download it](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
 
+Android Studio will provide you the Android SDK and emulator required to run and test your React
+Native apps.
+
+> Unless otherwise mentioned, keep all the setup defaults in tact. For example, the
```
Comment (vjeux) [inline docs/GettingStarted.md]: intact (one word)

File: docs/GettingStarted.md [hunk 3]
```diff
@@ -692,6 +702,7 @@ Now that you successfully started the project, let's modify it:
 
 <block class="mac android" />
 
+- Open `index.android.js` in your text editor of choice (e.g. [Nuclide](http://nuclide.io/docs/platforms/react-native/)) and edit some lines.
 - Press the `R` key twice **OR** open the menu (F2 by default, or ⌘-M in Genymotion) and select Reload JS to see your change!
 - Run `adb logcat *:S ReactNative:V ReactNativeJS:V` in a terminal to see your app's logs
```
Comment (vjeux) [inline docs/GettingStarted.md]: omg, we need some wrapper around this. This line is crazy :p

**Comment by facebook-github-bot-0** [comment]
@JoelMarcey updated the pull request.

**Comment by facebook-github-bot-7** [comment]
@JoelMarcey updated the pull request.

**Comment by vjeux** [comment]
Feel free to shipit

File: docs/GettingStarted.md (hunk 3 above)
Comment (JoelMarcey) [inline docs/GettingStarted.md]: Crazy how? It is an item in a bulleted list.

Comment (vjeux) [inline docs/GettingStarted.md]: Sorry, this is a more general thought. We shouldn't have people remember to write

```
adb logcat *:S ReactNative:V ReactNativeJS:V`
```

in order to see the logs. This is a pretty terrible developer experience :p

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by JoelMarcey** [comment]
@facebook-github-bot import

**Comment by facebook-github-bot-2** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/556978611128233/int_phab) to review.

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**